### ISSUE TITLE
Timeline

### ISSUE THREAD

**Issue by AABoyles** [CDCgov/MicrobeTrace#44]
Instead of the current, smoothing-based timeline, let's draw a block-based timeline. Here's what I'm imagining:
* A view that's designed to be docked across the bottom of the screen, like a media play bar or the timeline from MicroReact
* Each node gets a "block", which is an SVG rect, sitting on the point where their timestamp is. Multiple blocks on the same timestamp get stacked. Horizontally-overlapping blocks get drawn over.
* A "current time" bar allows the user to "play" through the timeline over around 10 seconds. Each time the bar crosses a timestamp which contains a node, fire `node_visibility` and make those nodes appear in the other views (like MicroReact).

**Comment by AABoyles** [comment]
See http://halftone.co/projects/self-immolations/ for a great example

**Comment by AABoyles** [comment]
Change of plans: instead of the node-based version we've developed so far, let's draw the epi curve instead.

**Comment by jaywokim** [comment]
-attach triggers to components to react to updates to node visibility updated through timeline
-spawn the node in a central location when running timeline in normal mode

**Comment by mossy426-cdc** [comment]
@AABoyles  Is this a performance optimization suggestion?  Not too sure of what the outcome of doing this will be.

**Comment by AABoyles** [comment]
No, it was a different visualization. I think it's basically OBE at this point, would archive.
